course/models.py

```diff
@@ -189,13 +190,23 @@ def namegen():
 post_save.connect(pseudonymize, sender=Enrollment)
 
 
+class UserTagManager(models.Manager): 
+    def set(self, **kwargs):
+        return self.get_or_create(**kwargs)
+
+    def unset(self, **kwargs):
+        self.filter(**kwargs).delete()
+
+
 class UserTag(UrlMixin, ColorTag):
     course_instance = models.ForeignKey('CourseInstance', related_name="usertags", on_delete=models.CASCADE)
     visible_to_students = models.BooleanField(default=False)
 
     class Meta:
         ordering = ['course_instance', 'name']
 
+    objects = UserTagManager()
```

Comment:
If you order this in fields, `objects`, and `Meta`, then it should match rest of the models.

Reply:
Done